src/components/Alert/Alert.styled.js

```diff
@@ -1,7 +1,23 @@
-import styled from 'styled-components';
+import styled from "styled-components";
 
 const StyledAlert = styled.div`
-    display: block;
-`
+  display: block;
+  background-color: #cce5ff;
+  border: 1px solid #b8daff;
+  border-radius: 4px;
+  width: 100%;
+  height: 50px;
```

Comment:
Ustawianie konkretnej wysokości to zły pomysł.
Co jeśli tekst nie będzie się mieścił w jednej linii?
Odstępy lepiej zrobić paddingiem.